playbooks/roles/nghttpx/vars/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+nghttpx_path: "/etc/nghttpx"
+nghttpx_config_file: "{{ nghttpx_path }}/nghttpx.conf"
+
+nghttpx_days_valid: "1825"
+
+# The package name list for Ubuntu 16.04 is from official README.
+# But all we need is nghttpx, so we could drop some of them (commented out).
+nghttpx_dependencies:
+  - g++
+  - make
+  - binutils
+  - autoconf
+  - automake
+  - autotools-dev
+  - libtool
+  - pkg-config
+  - zlib1g-dev
+  - libcunit1-dev
+  - libssl-dev
+  #- libxml2-dev
+  - libev-dev
+  #- libevent-dev
+  #- libjansson-dev
+  - libc-ares-dev
+  - libjemalloc-dev
+  - libsystemd-dev
+  #- libspdylay-dev
+  #- cython
+  #- python3-dev
+  #- python-setuptools
+  - gnutls-bin  # certtool
+
```

Comment:
I might be misunderstanding and this is a byproduct of being a WIP not intended to be merged but we wouldn't merge commented out code to master. Comments should be written for humans not disabled code :-)